packages/react-portal/src/components/Portal/Portal.types.ts

```diff
@@ -9,7 +9,7 @@ type PortalCommons = {
    * Where the portal children are mounted on DOM
    * @defaultValue a new element on document.body without any styling
    */
-  mountNode: HTMLDivElement | undefined;
+  mountNode: HTMLElement | undefined;
```

Comment:
```suggestion
  mountNode: HTMLElement | null;
```

this undefined should be null